client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,148 @@
+        alignItems="flex-start"
+        className={classes.root}
+      >
+        <Grid item xs={12} sm={7}>
+          <Paper elevation={3}>
+            <Box className={classes.media}>
+              <CardMedia className={classes.coverPhoto} image={house} title="Contemplative Reptile" />
+              <Avatar aria-label="sitter-photo" alt="Person" src={person} className={classes.profilePhoto} />
+            </Box>
+            <Card className={classes.profileDetail}>
+              <CardContent>
+                <Typography gutterBottom variant="h5" align="center">
+                  Norma Byers
+                </Typography>
+                <Typography paragraph variant="body1" color="textSecondary" align="center">
+                  Loving pet sitter
+                </Typography>
+                <Typography variant="body1" color="textSecondary" align="center">
+                  <LocationOnIcon color="secondary" className={classes.locationIcon} />
+                  Toronto, Ontario
+                </Typography>
+                <Typography gutterBottom variant="h6" className={classes.title}>
+                  About me
+                </Typography>
+                <Typography paragraph variant="body1" color="textPrimary">
+                  Animals are my passion! I will look after your pets with loving care. I have some availability for pet
+                  care in my home ase well. I have 10 yrs experience at the Animal Hospital, and have owned multiple
+                  pets for may years, including numerous rescues. Kindly email, text or call me and I will respond
+                  promptly!
+                </Typography>
+                <ImageList rowHeight={160} cols={4}>
+                  <ImageListItem cols={1}>
+                    <img src={dog} alt="dog" />
+                  </ImageListItem>
+                </ImageList>
+              </CardContent>
+            </Card>
+          </Paper>
+        </Grid>
+        <Grid item xs={12} sm={5}>
+          <Paper elevation={3}>
+            <Card className={classes.request}>
+              <Box>
+                <Typography variant="h6" align="center" className={classes.title}>
+                  $14/hr
+                </Typography>
+                <Rating name="size-large" defaultValue={4} size="large" className={classes.rating} readOnly />
+              </Box>
+              <Box className={classes.datesBox}>
+                <Typography className={classes.datesTitle}>drop in</Typography>
+                <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                  <KeyboardDatePicker
+                    autoOk
+                    disablePast
+                    variant="inline"
+                    inputVariant="outlined"
+                    format="dd MMMM yyyy"
+                    value={selectedDate}
+                    InputAdornmentProps={{ position: 'start' }}
+                    onChange={(date) => handleDateChange(date)}
```

Comment:
```suggestion
                    onChange={handleDateChange}
```
You can simplify this 🙌🏼 (we should apply to all occurances 👍🏼 )